src/components/RengoManagementPane/RengoManagementPane.tsx

```diff
@@ -20,7 +20,7 @@ import * as React from "react";
 import { _, pgettext } from "translate";
 
 interface RengoManagementPaneProperties {
-    user_id: number;
+    user: any;
```

Comment:
Hated to see a new `any` added, so made a PR for an interface!

https://github.com/GreenAsJade/online-go.com/pull/4